front/app/modules/commercial/analytics/admin/containers/Visitors/index.tsx

```diff
@@ -36,25 +33,14 @@ const query: Query = {
 };
 
 const Visitors = () => {
-  const { data: appConfig } = useAppConfiguration();
   const { data: authUser } = useAuthUser();
-
   const { data: analytics } = useAnalytics<Response>(
     query,
     undefined,
     authUser ? isAdmin(authUser) : false
   );
-  const visitorsDashboardEnabled = useFeatureFlag({
-    name: 'visitors_dashboard',
-  });
 
-  if (
-    !visitorsDashboardEnabled ||
-    !appConfig ||
```

Comment:
appConfig is not used, you shouldn't ever get to the admin if authUser is not defined (+ already checked via isAdmin)